src/main/resources/config.properties

```diff
@@ -16,4 +16,8 @@ Builder.matlab.modelcoverage.support.warning = To generate a Cobertura model cov
 Builder.matlab.exportstmresults.support.warning = To export Simulink Test Manager results, use MATLAB R2019a or a newer release.
 Builder.matlab.runner.script.target.file.linux.name = run_matlab_command.sh
 Builder.matlab.runner.script.target.file.windows.name = run_matlab_command.bat
-build.workspace.computer.not.found = Unable to access the computer for this build.
\ No newline at end of file
+build.workspace.computer.not.found = Unable to access the computer for this build.
+matlab.command.build.step.name = runMATLABCommand
+matlab.tests.build.step.name = runMATLABTests
+matlab.command.step.display.name = Runs MATLAB command and scripts
+matlab.tests.step.display.name = Runs MATLAB tests and produce test artifacts
```

Comment:
@mw-hrastega these are the changes you may review

Reply:
I revised the two statements that you have added. How/where is the text displayed?

Reply:
The text is displayed on Blue ocean pipeline plugin . as shown below 
![image](https://user-images.githubusercontent.com/47204011/86331876-50b64f80-bc67-11ea-9c2c-cb7ec1b863d3.png)